src/icons.scss

```diff
@@ -441,6 +447,11 @@
       @extend .fa-server;
       @extend .text-danger;
     }
+    &-disabled {
+      @extend .fa;
+      @extend .fa-server;
+      @extend .text-warning;
+    }
```

Comment:
Why 2 icons, `disabled` and `host-disabled`?
What's the difference between them and when should they be used?